baggage/HTTP_HEADER_FORMAT.md

```diff
@@ -101,3 +102,17 @@ For example, if you have non-production requests that flow through the same serv
 ```
 baggage: isProduction=false
 ```
+# Mutating baggage
+A system receiving a `baggage` request header SHOULD send it to outgoing requests.
+A system MAY mutate the value of this header before passing it on.
+
+As the producer and consumer of `baggage` SHOULD be within the same logical system and the content of the
+key/value pairs is opaque to the spec, additional proprietary rules for mutations MAY be defined.
+Example: Defining a precedence when deduplicating keys.
```

Comment:
Using SHOULD here seems odd. Normatively suggesting the producer and consumer need to be the same system is out of scope I think.

```suggestion
Because baggage entry keys, values, and metadata are not specified here, producers and consumers MAY agree on any set of mutation rules that don't violate the specification. For example, keys may be deduplicated by keeping the first entry, keeping the last entry, or concatenating values together.
```